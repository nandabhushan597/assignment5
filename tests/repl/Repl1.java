12
  12e4
  true
  false
  "hi there"
  null
  :type 12
  :type 12l
  :type .14f
  :type null
  :type "hi"
  :type '\''
  :quit
  
