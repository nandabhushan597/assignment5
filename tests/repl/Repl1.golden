Exit code: ExitSuccess
Stdout:
Welcome to the Java interpreter.
Type `:?` at the prompt for the list of commands.
Java> 12
Java> 120000.0
Java> true
Java> false
Java> @2 => "hi there"
Java> null
Java> int
Java> long
Java> float
Java> <nulltype>
Java> String
Java> char
Java> Good-bye.

Stderr:
